cgi/product_multilingual.pl

```diff
@@ -805,6 +805,154 @@ ($product_ref, $field, $language, $request_ref)
 	return $html_field;
 }
 
+=head2 get_nutrient_level ( $nutriment )
+
+Determines the level of a nutrient based on the number of leading dashes.
+
+The level is used to establish whether a nutrient is a top-level nutrient or a sub-nutrient (e.g., fat -> saturated fat -> omega-3).
+
+=head3 Arguments
+
+=head4 Nutrient $nutriment
+
+The nutrient identifier string, which may contain leading dashes to denote levels.
+
+=head3 Return values
+
+Returns an integer indicating the nutrient's level.
+- 0 for main nutrients
+- 1 for sub-nutrients
+- 2 for sub-sub-nutrients, etc.
+
+=cut
+
+sub get_nutrient_level {
```

Comment:
We use signatures to pass function parameters:

```suggestion
sub get_nutrient_level($nutriment) {
```

And then you remove the line my ($nutriment) = @_;